appengine/standard/endpoints-frameworks-v2/iata/requirements.txt

```diff
@@ -1,2 +1,3 @@
 google-endpoints==4.8.0
 google-endpoints-api-management==1.11.1
+rsa==4.0
```

Comment:
I think 4.5 is the last version with 2.7 compatibility, if that's the only issue here: https://github.com/sybrenstuvel/python-rsa/blob/main/CHANGELOG.md#version-43--45---released-2020-06-12

Reply:
Done!